backend/apps/chats/models.py

```
@@ -0,0 +1,178 @@
+	)
+	created_at = models.DateTimeField(auto_now_add=True)
+	updated_at = models.DateTimeField(auto_now=True)
+
+	class Meta:
+		ordering = ["-updated_at"]
+		constraints = [
+			models.UniqueConstraint(
+				fields=["residence", "member_one", "member_two"],
+				name="uniq_private_conversation",
+			)
+		]
```

Ruff (0.15.5)

[warning] Mutable default value for class attribute

(RUF012)